server/RoboFile.php

```diff
@@ -43,7 +43,7 @@ public function deployPantheon($branchName = 'master') {
 
     if ($result->getMessage()) {
       $this->yell($result->getMessage());
-      throw new Exception('The GitHub working directory is dirty. Please commit any pending changes or add to .gitignore.');
+      // throw new Exception('The GitHub working directory is dirty. Please commit any pending changes or add to .gitignore.');
```

Comment:
May I ask what was the problem here? Normally it's a safeguard that helps us to prevent broken deployments from local machines.
On Travis, sometimes we need to tweak gitignore for that.

Reply:
@AronNovak 

I will revert this change. The problem is with `.gitignore`:

![Selection_2418](https://github.com/TIP-Global-Health/eheza-app/assets/17882245/419b7932-1404-4e0f-91df-9b51fd2dc6f0)